jenkins/pull_request_creator.py

```diff
@@ -76,12 +73,12 @@ def _create_new_branch(self):
 
     def _create_new_pull_request(self):
         # If there are reviewers to be added, split them into python lists
-        if isinstance(self.user_reviewers, (str, six.text_type)) and self.user_reviewers:
+        if isinstance(self.user_reviewers, (str, str)) and self.user_reviewers:
```

Comment:
This now checks to see if it's an instance of `str` or `str`; should switch it from a tuple to just `str`.  Same below.